### DISCUSSION TITLE
auto refresh dashboard embededd

### DISCUSSION THREAD

**Discussion by MarioBaselice** [apache/superset#22094] [Q&A / Help]
Is it possible refreshing dashboard in embededd ?
it work only in superset windows browser.

I m setting refresh in advanced properties as follow:
{
  "show_native_filters": true,
  "color_scheme": "googleCategory10c",
  "refresh_frequency": 10,
  "expanded_slices": {},
  "label_colors": {},
  "timed_refresh_immune_slices": [],
  "default_filters": "{}",
  "chart_configuration": {},
  "stagger_refresh": true
}

Thanks

**Comment by harry-pham-viact** [comment]
Same to me.
